data/core.yaml

```diff
@@ -219,14 +219,41 @@ en:
       multiple_ways: There are too many lines here to split.
       connected_to_hidden: This can't be split because it is connected to a hidden feature.
     restriction:
-      help:
-        select: Click to select a road segment.
-        toggle: Click to toggle turn restrictions.
-        toggle_on: 'Click to add a "{restriction}" restriction.'
-        toggle_off: 'Click to remove the "{restriction}" restriction.'
       annotation:
         create: Added a turn restriction
         delete: Deleted a turn restriction
+  restriction:
+    controls:
+      distance: Distance
+      distance_up_to: "Up to {distance}"
+      via: Via
+      via_node_only: "Node only"
+      via_up_to_one: "Up to 1 way"
+      via_up_to_multiple: "Up to {num} ways"
+    help:
+      no: NO
+      only: ONLY
+      allowed: Allowed
+      indirect: "(indirect)"
+      turn:
+        left_turn: Left Turn
+        right_turn: Right Turn
+        u_turn: U-Turn
+        straight_on: Straight On
+      no_turn_string: "{no} {turn} {indirect}"
+      only_turn_string: "{only} {turn} {indirect}"
+      allowed_turn_string: "{turn} {allowed} {indirect}"
+      from: FROM
+      via: VIA
+      to: TO
+      from_name: "{from} {fromName}"
+      from_name_to_name: "{from} {fromName} {to} {toName}"
```

Comment:
Languages with declension rules won’t be able to translate this well, but I guess the fact that the prepositions are **bold** makes it feel more like a form and less like a coherent sentence anyways.

Reply:
Yeah this is extra tricky because the `{fromName}` and `{toName}` might not actually be proper named roads, but rather strings like "Residential Road" or "Service Road" which could follow different rules depending on the modifier.  We don't support different conjugation of the preset name strings, so I'll punt on this one for now.  (But yeah, hoping the way we present it in the UI makes it seem more like a filled in form than a grammatically incorrect sentence).